### PR TITLE
Feature - QOL - Prettier Bash Prompt

### PR DESCRIPTION
**Pull request by RossMurr4y**
## Description

Implemented a quality-of-life change to the container bash prompt, placing the current directory on its own line and making clear which branch of a git repo the user is currently viewing.

## Motivation and Context

The primary use of the container is to work with hamlet CMDBs which are (typically) stored under git version control. Because hamlet operations are dependant on the working directory, often the current working directory is a particularly lengthy string. This can make it difficult to read output at times, particularly when debugging. It also means even short commands get word-wrapped within a terminal session which is similarly difficult to read.

### Before (click to embiggen)
before-prettier

### After (click to embiggen)
after-prettier

## How Has This Been Tested?

Local image build

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Breaking Actions

N/A

## Checklist:

- [ ] My change requires a change to the [documentation](https://github.com/hamlet-io/docs).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] None of the above required

### CODE REVIEW

File: utilities/hamlet/Dockerfile [hunk 1]
```diff
@@ -41,7 +41,9 @@ RUN chmod -R u+rwx /build/scripts && \
 
 RUN echo "Cloning images on $(date +%s)"
 RUN /build/scripts/build_hamlet.sh && \
-    /build/scripts/clone.sh
+    /build/scripts/clone.sh && \
+    echo "Updating Shell Prompt" && \
+    /build/scripts/shellprompt.sh
```
Comment (roleyfoley) [inline utilities/hamlet/Dockerfile]: Might be better to have this in
https://github.com/hamlet-io/docker-hamlet/blob/master/images/stretch/Dockerfile 

The utilities one is more around building hamlet on an image. For example apling doesn't use bash so this script might not work on there.

Comment (RossMurr4y) [reply utilities/hamlet/Dockerfile]: Ahh, I hadn't considered someone overwriting the ARG and specifying a different underlying image. Makes sense though.

I'll update this, then happy to merge?

Comment (RossMurr4y) [reply utilities/hamlet/Dockerfile]: The hamlet user is created in the utility script. I could implement it for root user in base image as you suggest and then in hamlet image perform a check to see if using bash, then `cp /root/.bashrc /home/hamlet/.bashrc` ?

Comment (roleyfoley) [reply utilities/hamlet/Dockerfile]: Yep that works

Comment (RossMurr4y) [reply utilities/hamlet/Dockerfile]: updated and rebased. Ready for re-review.

**Comment by RossMurr4y** [comment]
this is to be updated and replaced by hamlet-io/docker-hamlet#21

**Pull request merged**